### PR TITLE
completed week3 node.js assignment

### PR DESCRIPTION
**Pull request by shreyansh-khatri**
Pull Request Questionnaire

1) Briefly describe your change.
Completed NodeJs week3 assignments.

2) What parts of the application are affected by your change?
My changes only affected the week3 branch of this repository.

3) How did you test your change?
I tested various parts such as validation, different inputs and some other edge cases also used JEST to test code.

### CODE REVIEW

**Review by choudhary-keshav — APPROVED** [review]

File: src/Week3/assignment1.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  databaseURL: "https://your-project-id.firebaseio.com",
+});
+
+const db: admin.firestore.Firestore = admin.firestore();
+app.use(bodyParser.json());
+
+app.get("/users", async (req: Request, res: Response): Promise<void> => {
+  try {
+    const snapshot: QuerySnapshot = await db.collection("users").get();
+    const users: any[] = [];
```
Comment (aditya-s-19) [inline src/Week3/assignment1.ts:26]: try to avoid any

**Review by aditya-s-19 — APPROVED** [review]

**Review by Harrrrry — APPROVED** [review]
